src/main/java/wooteco/subway/path/domain/policy/discountpolicy/ChildrenDiscountPolicy.java

```diff
@@ -0,0 +1,29 @@
+package wooteco.subway.path.domain.policy.discountpolicy;
```

Comment:
```suggestion
package wooteco.subway.path.domain.policy.discount;
```
policy 패키지 내부에 있기 때문에 discountpolicy의 policy는 빼도 될 것 같아요 ^_^